packages/react-native-client/ios/Utils/PermissionUtils.swift

```diff
@@ -0,0 +1,17 @@
+import AVFoundation
+
+class PermissionUtils {
+    static func requestCameraPermission() async -> Bool { await requestAccessIfNeeded(for: .video) }
+    static func requestMicrophonePermission() async -> Bool { await requestAccessIfNeeded(for: .audio) }
+    
+    private static func requestAccessIfNeeded(for mediaType: AVMediaType) async -> Bool {
+        let status = AVCaptureDevice.authorizationStatus(for: mediaType)
+        
+        var isAuthorized = status == .authorized
+        if status == .notDetermined {
+            isAuthorized = await AVCaptureDevice.requestAccess(for: mediaType)
+        }
+        
+        return isAuthorized
```

Comment:
```suggestion
        return if status == .notDetermined {
            await AVCaptureDevice.requestAccess(for: mediaType)
        } else {
            status == .authorized
        }
```

Reply:
Why? Status won't be updated after `requestAccess` call.